components/About.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  Box,
+  Heading,
+  Link,
+  ListItem,
+  Stack,
+  Text,
+  UnorderedList,
+  Container,
+  Center,
+} from "@chakra-ui/react";
+import React from "react";
+
+const About = () => {
+  return (
+    <Container maxW="container.md" className="coc-container">
```

Comment:
I'd just added this class name when I was fiddling with the iFrame for the Google form, you can remove it here.